requirements/check-types.txt

```diff
@@ -1,2 +1 @@
 mypy
-django-stubs[compatible-mypy]
```

Comment:
You removed this from the requirements file. I think this is causing the issue.

Reply:
The underlying problem is that, without this, `View` is treated as `Any` and thus causes the signature overlap.